src/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+
+}
+
+.stars {
+  display: flex;
+
+}
```

Comment:
- From the checklist: Each BEM block has its own separate file
- Don't leave unnecessary blank lines